service/extensions/extensions.go

```diff
@@ -46,6 +47,7 @@ func (bes *Extensions) Start(ctx context.Context, host component.Host) error {
 				instanceID,
 				component.NewPermanentErrorEvent(err),
 			)
+			extLogger.WithOptions(zap.AddStacktrace(zap.DPanicLevel)).Error("Failed to start extension", zap.Error(err))
```

Comment:
Can we add a comment about the 'DPanicLevel'?

Reply:
Done.